Src/CSharpier.Tests/DocPrinterTests.cs

```diff
@@ -7,7 +7,7 @@ namespace CSharpier.Tests
 {
     public class DocPrinterTests
     {
-        private static string NewLine = System.Environment.NewLine;
+        private static readonly string NewLine = System.Environment.NewLine;
```

Comment:
instead of `static readonly`, why can't you just do `const`, since they are strings?

Reply:
`Environment.NewLine` isn't a constant value, it would vary between windows and linux